packages/admin-ui-form/src/watch/use-watch.ts

```diff
@@ -0,0 +1,43 @@
+import { useWatch as useWatchBase } from 'react-hook-form'
+import type { FormState } from '../form'
+
+/**
+ * React Hook for subscribe to input changes
+ * @example
+ * const form = useFormState()
+ * const subscribedValue = useWatch({
+ *   form,
+ *   name: "fieldName"
+ * })
+ */
+export function useWatch(props: UseWatchProps) {
+  const { form, name, defaultValue, disabled } = props
+
+  return useWatchBase({
+    control: form.control,
+    name,
+    defaultValue,
+    disabled,
+  })
+}
+
+export interface UseWatchProps {
+  /** Target form */
+  form: FormState
+  /**  Name of the field */
+  name: string
+  /** Default value for useWatch to return before the initial render */
+  defaultValue?: unknown
+  /**
+   * Option to disable the subscription
+   * @default false
+   * */
+  disabled?: boolean
+  /**
+   * Enables an exact match for input name subscriptions
+   * @default false
+   */
+  exact?: boolean
```

Comment:
Will this prop be used in the `useWatchBaseHook`?

Reply:
Ops saw after merge. Probably yes, it will be used.